### PR TITLE
cpu and memory requests for source_container in reactor config map

### PR DESCRIPTION
**Pull request by rcerven**
* CLOUDBLD-4118

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [n/a] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

**Comment by ben-alkov** [comment]
@rcerven; Doc update?

**Comment by rcerven** [comment]
> @rcerven; Doc update?

not sure which doc update, for such a server side reactor-config-map changes we usually just refer in docs to schema

also not sure yet if somebody would like to have it in different section

**Comment by rcerven** [comment]
related osbs-client PR https://github.com/containerbuildsystem/osbs-client/pull/1012

**Review by mkosiarc — APPROVED** [review]
Not sure what to do with the docs here or in related mr in osbs-client. but other that that approved

**Comment by ben-alkov** [comment]
> for such a server side reactor-config-map changes we usually just refer in docs to schema

ACK, I didn't know this.

**Review by ben-alkov — APPROVED** [review]

**Review by tkdchen — APPROVED** [review]

File: atomic_reactor/schemas/config.json [hunk 1]
```diff
@@ -674,6 +674,14 @@
...
               "description": "Url with denylist yaml file, which will be used to exclude sources from cachito ",
               "type": "string"
+          },
+          "cpu_request": {
+              "description": "Openshift cpu request for build",
+              "type": "string"
+          },
+          "memory_request": {
+              "description": "Openshift memory request for build",
+              "type": "string"
```
Comment (MartinBasti) [inline atomic_reactor/schemas/config.json:684]: Didn't we agreed to always add examples to schema?

Comment (rcerven) [reply atomic_reactor/schemas/config.json]: added

**Review by MartinBasti — APPROVED** [review]

**Comment by rcerven** [comment]
rn updated: cpu and memory requests for source_container in reactor config map

**Pull request merged**